baize/asgi/websocket.py

```diff
@@ -8,8 +8,9 @@
 
 
 class WebSocketDisconnect(Exception):
-    def __init__(self, code: int = 1000) -> None:
```

Comment:
This doesn't seem to be related to Range Request? If you have an idea for modifying this object, we can open another PR to discuss the modification.

Reply:
I might have been mistaken, but I thought a test was failing because it was trying to pass something to WebSocketDisconnect. I can remove the change.